docs/marionette.region.md

```diff
@@ -381,98 +308,3 @@ var MyView = Mn.View.extend({
   }
 });
 ```
-
-## Region Events And Callbacks
-
-A region will raise a few events on itself and on the target view when showing and destroying views.
-
-### Events Raised on the Region During `show()`
-
-* `before:show` / `onBeforeShow` - Called after the view has been rendered, but before its been displayed.
-* `show` / `onShow` - Called when the view has been rendered and displayed.
-* `before:empty` / `onBeforeEmpty` - Called before the view has been emptied.
-* `empty` / `onEmpty` - Called when the view has been emptied.
-
-```javascript
-view.supportsRenderLifecycle = true;
-view.supportsDestroyLifecycle = true;
-```
-
-## Custom Region Classes
-
-You can define a custom region by extending from
-`Region`. This allows you to create new functionality,
-or provide a base set of functionality for your app.
-
-### Attaching Custom Region Classes
-
-Once you define a region class, you can attach the
-new region class by specifying the region class as the
-value. In this case, `addRegions` expects the constructor itself, not an instance.
-
-```javascript
-var Mn = require('backbone.marionette');
-
-var FooterRegion = Mn.Region.extend({
-  el: "#footer"
-});
-
-var MyView = Mn.View.extend({
-  regions: {
-    footerRegion: FooterRegion
-  }
-});
-```
-
-You can also specify a selector for the region by using
-an object literal for the configuration.
-
-```javascript
-var Mn = require('backbone.marionette');
-
-var FooterRegion = Mn.Region.extend({
-  el: "#footer"
-});
-
-var MyView = Mn.View.extend({
-  regions: {
-    footerRegion: {
-      regionClass: FooterRegion
-      el: "#footer",
-    }
-  }
-});
-```
-
-Note that a region must have an element to attach itself to. If you
-do not specify a selector when attaching the region instance to your
-Application or LayoutView, the region must provide an `el` either in its
-definition or constructor options.
-
-### Instantiate Your Own Region
```

Comment:
Behaviors are very contextual to the view so your view has to be aware of what is applied to it.  Regions are containers that can provide a view with context.  The difference while slight makes the code easier to organize I think.  Aside from transitions which would be hard with a behavior, in my app a few things change structurally between desktop / tablet / phone..  The views don't know what context they are in or care, but they make a request to be shown in the sidebar region.  On desktop that region is always visible and can be resized.  On tablet it is fixed horizontally and collapses vertically.  On mobile it is collapsed by default and a menu button slides it into view.  And I can put dozens of different views into the region on the three different platforms and the container handles what happens to them